frontend/src/modules/Settings/Components/Settings.tsx

```diff
@@ -0,0 +1,83 @@
+import { DropdownSelect } from '@core/index'
+import { Box, IconButton, SelectChangeEvent } from '@mui/material'
+import Menu from '@mui/material'
+import MenuItem from '@mui/material/MenuItem'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { DASHBOARD_PATH } from '@core/index'
+import { ReactComponent as LogoImg } from '@assets/img/lscicon.svg'
+import { AccountMenu } from 'Dashboard/Components/AccountMenu'
+
+export const Settings = () => {
+  const [currRoster, setCurrRoster] = useState<string>('SP23')
+
+  const changeCurrRoster = (event: SelectChangeEvent) => {
+    // function to only open the survey of the semester
+    setCurrRoster(event.target.value)
+  }
+  return (
+    <Box
+      sx={{ pl: '5rem', pr: '5rem', display: 'flex', flexDirection: 'column' }}
+    >
+      <Box
+        sx={{
+          width: '100%',
+          height: 'fit-content',
+          padding: '2.5rem',
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+        }}
+      >
+        <IconButton
+          color="secondary"
+          component={Link}
+          to={{
+            pathname: DASHBOARD_PATH,
+          }}
+          sx={{
+            border: 'none',
+            alignSelf: 'flex-start',
+          }}
+          disableRipple
+          disableFocusRipple
+        >
+          <LogoImg />
+        </IconButton>
+        <AccountMenu
+          selectedRoster={currRoster}
+          setSelectedRoster={setCurrRoster}
+          showMetricsLink={true}
+          showDashboardLink={true}
+          showSettingsLink={false}
+        />
+      </Box>
+      <Box
+        sx={{
+          pl: '5rem',
+          pr: '5rem',
+          display: 'grid',
+          alignItems: 'center',
+        }}
+      >
+        <Box sx={{ typography: 'h5' }}>Change Semester</Box>
+
+        <DropdownSelect
+          value={currRoster}
+          onChange={changeCurrRoster}
+          sx={{
+            alignContent: 'right',
+            right: '1px',
+          }}
+        >
+          <MenuItem value="SU22">Summer 2022</MenuItem>
```

Comment:
So, I believe Colin's PR will provide a backend endpoint to get the current and list of available semesters - the goal being that the semester "state" is stored in the backend and is global for all users. One thing you could do to to prepare for it is to create a list for now and then map through that list to render `MenuItems`.

Reply:
^ yeah eventually this is will be programmatic and not hardcoded so you can make this into some data array and render it in a map rather than hard coding it here now.